inc/ptw.h

```diff
@@ -0,0 +1,142 @@
+#ifndef PTW_H
+#define PTW_H
+
+#include "cache.h"
+#include <map>
+
+#define IS_PSCL5 11
+#define IS_PSCL4 12
+#define IS_PSCL3 13
+#define IS_PSCL2 14
+
+#define PSCL5_SET 1
+#define PSCL5_WAY 2
+
+#define PSCL4_SET 1
+#define PSCL4_WAY 4
+
+#define PSCL3_SET 2
+#define PSCL3_WAY 4
+
+#define PSCL2_SET 4
+#define PSCL2_WAY 8
+
+#define PTW_RQ_SIZE STLB_MSHR_SIZE
+#define PTW_WQ_SIZE 0
+#define PTW_MSHR_SIZE 1
+#define PTW_PQ_SIZE 0
+
+#define MMU_CACHE_LATENCY 1
```

Comment:
Consider including these as entries in the configuration file. This allows others to modify them easily as is appropriate for their research.